src/App/component/AddProjectDialog.jsx

```diff
@@ -159,4 +79,4 @@ export default function AddProjectDialog({open, reloadProjects, handleClose}) {
       </DialogActions>
     </Dialog>
   )
-}
+}
```

Comment:
Suggest left a new line at the end of a file.
```suggestion
}

```